### PR TITLE
New package: FiloSottile.mkcert version 1.4.4

### PR DESCRIPTION
**Pull request by cnshenj**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.2 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.2.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/80613)

### CODE REVIEW

File: manifests/f/FiloSottile/mkcert/1.4.4/FiloSottile.mkcert.defaultLocale.yml [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+Publisher: FiloSottile
+PublisherUrl: https://github.com/FiloSottile
+Author: Filippo Valsorda
+PackageName: mkcert
+PackageUrl: https://github.com/FiloSottile/mkcert
+License: BSD 3-Clause
+LicenseUrl: https://github.com/FiloSottile/mkcert/blob/master/LICENSE
+ShortDescription: mkcert is a simple tool for making locally-trusted development certificates.
+ManifestType: defaultLocale
+ManifestVersion: "1.2.0"
```
Comment (Trenly) [inline manifests/f/FiloSottile/mkcert/1.4.4/FiloSottile.mkcert.defaultLocale.yml:14]:
```suggestion
ManifestVersion: 1.2.0

```
Newline preferred at end of file

File: manifests/f/FiloSottile/mkcert/1.4.4/FiloSottile.mkcert.installer.yml [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+Installers:
+  - Architecture: x64
+    InstallerUrl: https://github.com/FiloSottile/mkcert/releases/download/v1.4.4/mkcert-v1.4.4-windows-amd64.exe
+    InstallerSha256: D2660B50A9ED59EADA480750561C96ABC2ED4C9A38C6A24D93E30E0977631398
+  - Architecture: arm64
+    InstallerUrl: https://github.com/FiloSottile/mkcert/releases/download/v1.4.4/mkcert-v1.4.4-windows-arm64.exe
+    InstallerSha256: 793747256C562622D40127C8080DF26ADD2FB44C50906CE9DB63B42A5280582E
+InstallerType: portable
+ManifestType: installer
+ManifestVersion: "1.2.0"
```
Comment (Trenly) [inline manifests/f/FiloSottile/mkcert/1.4.4/FiloSottile.mkcert.installer.yml:13]:
```suggestion
ManifestVersion: 1.2.0

```
Newline preferred at end of file

File: manifests/f/FiloSottile/mkcert/1.4.4/FiloSottile.mkcert.version.yml [hunk 3]
```diff
@@ -0,0 +1,6 @@
+# yaml-language-server: $schema=https://github.com/microsoft/winget-cli/raw/master/schemas/JSON/manifests/v1.2.0/manifest.version.1.2.0.json
+PackageIdentifier: FiloSottile.mkcert
+PackageVersion: 1.4.4
+DefaultLocale: en-US
+ManifestType: version
+ManifestVersion: "1.2.0"
```
Comment (Trenly) [inline manifests/f/FiloSottile/mkcert/1.4.4/FiloSottile.mkcert.version.yml:6]:
```suggestion
ManifestVersion: 1.2.0

```
Newline preferred at end of file

File: manifests/f/FiloSottile/mkcert/1.4.4/FiloSottile.mkcert.installer.yml [hunk 4]
```diff
@@ -0,0 +1,13 @@
+# yaml-language-server: $schema=https://github.com/microsoft/winget-cli/raw/master/schemas/JSON/manifests/v1.2.0/manifest.installer.1.2.0.json
+PackageIdentifier: FiloSottile.mkcert
+PackageVersion: 1.4.4
+Installers:
+  - Architecture: x64
+    InstallerUrl: https://github.com/FiloSottile/mkcert/releases/download/v1.4.4/mkcert-v1.4.4-windows-amd64.exe
+    InstallerSha256: D2660B50A9ED59EADA480750561C96ABC2ED4C9A38C6A24D93E30E0977631398
+  - Architecture: arm64
+    InstallerUrl: https://github.com/FiloSottile/mkcert/releases/download/v1.4.4/mkcert-v1.4.4-windows-arm64.exe
+    InstallerSha256: 793747256C562622D40127C8080DF26ADD2FB44C50906CE9DB63B42A5280582E
```
Comment (Trenly) [inline manifests/f/FiloSottile/mkcert/1.4.4/FiloSottile.mkcert.installer.yml:10]:
```suggestion
    InstallerSha256: 793747256C562622D40127C8080DF26ADD2FB44C50906CE9DB63B42A5280582E
    Commands:
    - mkcert
```

**Review by Trenly — CHANGES_REQUESTED** [review]

**Review by ItzLevvie — APPROVED** [review]
It looks like the file names in this pull request are incorrect.

1. `FiloSottile.mkcert.version.yml` needs to be `FiloSottile.mkcert.yaml`
2. `FiloSottile.mkcert.defaultLocale.yml` needs to be `FiloSottile.mkcert.locale.en-US.yaml`
3. `FiloSottile.mkcert.installer.yml` needs to be `FiloSottile.mkcert.installer.yaml`

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
Please see https://github.com/microsoft/winget-pkgs/pull/80613#pullrequestreview-1126946105

**Comment by cnshenj** [comment]
> Please see [#80613 (review)](https://github.com/microsoft/winget-pkgs/pull/80613#pullrequestreview-1126946105)

It this really required? If so, shouldn't `winget validate` detect it instead of letting reviewer providing feedback? `winget validate` passes and successfully installs the package - this means `winget` actually has code to read `.yml` files.

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
> It this really required?

Yes - it's required for the WinGet package repository.

WinGet will able to validate `.YML` files but the repository (as well as the Validation Pipelines) now focuses on `.YAML` files to ensure consistency between file types (as well as the capitalization of the file types).

winget-create / YamlCreate.ps1 will allow you to generate manifest files based on what the repository accepts.

**Comment by cnshenj** [comment]
> > It this really required?
> 
> Yes - it's required for the WinGet package repository.
> 
> WinGet will able to validate `.YML` files but the repository (as well as the Validation Pipelines) now focuses on `.YAML` files to ensure consistency between file types (as well as the capitalization of the file types).
> 
> winget-create / YamlCreate.ps1 will allow you to generate manifest files based on what the repository accepts.

Files renamed.

This requirement should be incorporated into the regular validation to save contributors/reviewers' time.

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
Now you need to rename the following files:
1. `FiloSottile.mkcert.version.yaml` needs to be `FiloSottile.mkcert.yaml`
2. `FiloSottile.mkcert.defaultLocale.yaml` needs to be `FiloSottile.mkcert.locale.en-US.yaml`

**Review by ItzLevvie — APPROVED** [review]

**Comment by ItzLevvie** [comment]
Close with reason: To be replaced with a new PR;

**Comment by ItzLevvie** [comment]
> This requirement should be incorporated into the regular validation to save contributors/reviewers' time.

It's usually displayed in https://github.com/microsoft/winget-pkgs/pull/80613/checks?check_run_id=8684095056 whenever there's an issue with your pull request.

Before winget-create & YamlCreate.ps1 existed people were creating manifests by hand which created a lot of inconsistencies such as spelling mistakes, capitalization issues, etc.

Now that winget-create & YamlCreate.ps1 exists people should take advantage of one of the tools mentioned so you're less likely to have manifest submission issues as you're less prone to make mistakes.

**Pull request closed without merging**